### PR TITLE
[Speech] Update bindings for XCode 13.0 beta 1

### PR DESCRIPTION
**Pull request by rachelkang**

### CODE REVIEW

File: tests/xtro-sharpie/iOS-Speech.todo [hunk 1]
```diff
@@ -1,2 +0,0 @@
-!deprecated-attribute-missing! SFSpeechRecognitionRequest::interactionIdentifier missing a [Deprecated] attribute
-!deprecated-attribute-missing! SFSpeechRecognitionRequest::setInteractionIdentifier: missing a [Deprecated] attribute
```
Comment (tj-devel709) [inline tests/xtro-sharpie/iOS-Speech.todo:2]: I think you should be able to delete this file if there is not anything else in there!

**Review by tj-devel709 — APPROVED** [review]

**Review by mandel-macaque — CHANGES_REQUESTED** [review]
As @tj-devel709 mentioned, empty files should be removed from xtro.

File: tests/xtro-sharpie/iOS-Speech.todo (hunk 1 above)
Comment (rachelkang) [reply tests/xtro-sharpie/iOS-Speech.todo:2]: I had no idea - thanks for catching this!

**Review by mandel-macaque — APPROVED** [review]

**Pull request merged**